src/data/malamud_dataset.py

```diff
@@ -0,0 +1,11 @@
+import gensim
+import connectorx as cx
+
+class MalamudDataset:
+    def __init__(self, postgres_uri: str):
+        self.df = cx.read_sql(postgres_uri, "SELECT * FROM docs.doc_ngrams_0")
+        self.ngrams = self.df.ngram.apply(gensim.utils.simple_preprocess)
```

Comment:
I wonder if we will have a problem storing all of these in memory eventually. Any thoughts on this or ways to circumvent it?

Reply:
@christianjans Should be fixed now! I added a `chunk_size` param so we can experiment with different numbers of rows to see what's fastest. Currently it's training with chunk_size = 1M and the memory holds steady